axelrod/strategies/cycler.py

```diff
@@ -72,9 +72,11 @@ def __init__(self, cycle="CCD") -> None:
         self.classifier['memory_depth'] = len(cycle) - 1
 
     def strategy(self, opponent: Player) -> Action:
+        actions = {'C': Actions.C, 'D': Actions.D}
```

Comment:
yes.  i saw the issue where he mentioned making Actions an Enum, and had that in mind.